cluster/install.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+####################################################################
+# Base setup for a cluster with 2 nodes hosting the application pods
+####################################################################
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+#
+# Create a KIND cluster
+#
+kind delete cluster --name=curity 2>/dev/null
+kind create cluster --name=curity --config=./cluster.yaml
```

Comment:
`cluster.yaml` contains twice the entry `- role: worker`

Reply:
This means 2 worker nodes that host pods - it is a nice feature of KIND that you can do this - eg I don't think you can do so in minikube.